### PR TITLE
Custom cell templates

### PR DESCRIPTION
**Pull request by klumba12**

### CODE REVIEW

File: demo/pages/cell/index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<q-grid columns="$ctrl.columns"
+		  rows="$ctrl.rows">
+	<q-grid:body>
+		<q-grid:td key="name.last">
+			<b>{{$ctrl.$value()}}</b>
```
Comment (klumba12) [inline demo/pages/cell/index.html]: need to get rid of $ctrl

File: src/core/infrastructure/guard.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+
+/**
+ * If value is null or undefined exception will be thrown
+ * @param {any} value - Value to check
+ * @param {string} name - Argument name
+ */
+export function notNull(value, name){
+	if (value === null || isUndefined(value)) {
```
Comment (titovmx) [inline src/core/infrastructure/guard.js:22]: May adding isNull function make sense?

File: src/core/infrastructure/guard.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import Error from  './error';
+import {isUndefined} from '../services/utility';
+
```
Comment (titovmx) [inline src/core/infrastructure/guard.js:3]: Can you show me where it's using?

Comment (klumba12) [reply src/core/infrastructure/guard.js:3]: what do you mean? isUndefined used in guard e.g.

File: src/core/infrastructure/guard.js (hunk 2 above)
Comment (klumba12) [reply src/core/infrastructure/guard.js:22]: I'm not sure, we'll try to write in "not undedined" style, however maybe we can do it

File: src/core/infrastructure/guard.js (hunk 3 above)
Comment (titovmx) [reply src/core/infrastructure/guard.js:3]: I mean guard exactly :)

**Review by titovmx — APPROVED** [review]

**Pull request merged**